aiida/workflows/arithmetic/multiply_add.py

```diff
@@ -32,6 +32,7 @@ def define(cls, spec):
         spec.input('x', valid_type=Int)
         spec.input('y', valid_type=Int)
         spec.input('z', valid_type=Int)
+        spec.input('t', valid_type=Int, required=False, default=Int(0))
```

Comment:
I wouldn't change this workflow as it is used for documentation purposes and adding this can unnecessarily confuse users. You say you added it for testing, but I don't see any tests?

Reply:
I wouldn't add another workchain just to test this, definitely don't need to add an entire workchain and register it with normal entry points just for tests. You can also just write the workchain in the test file itself, such as is done in `tests/engine/test_work_chain.py` for example.

Even better though, if you really want to add a test for this, I would simply instantiate an instance of _any_ workchain and then simply set `self._awaitables` to contain an awaitable and set the process to be paused. Then you call `_update_process_status` and verify that the `_pre_paused_status` attribute is correctly set to the new status.

Reply:
I am not going to add a test for this. But, thanks for the suggestions, this is helpful for my other use case.